tlm_teamd/main.cpp

```diff
@@ -108,7 +108,10 @@ int main()
             else if (res == swss::Select::TIMEOUT)
             {
                 teamdctl_mgr.process_add_queue();
-                values_store.update(teamdctl_mgr.get_dumps());
+                // In the case of lag removal, there is a scenario where the select::TIMEOUT
+		// occurs, it triggers get_dumps incorrectly for resource which was in process of 
+		// getting deleted. The fix here is to retry and check if this is a real failure.
```

Comment:
please convert the tab to spaces

Reply:
@lguohan updated as per comments, could you review.